### PR TITLE
MyEtherWallet => MyCrypto

### PR DESCRIPTION
**Pull request by clrxbl**
Note: Knowledge base links in translations are not working due to the new knowledge base & some links still link to the MyEtherWallet Chrome Extension.

### CODE REVIEW

File: common/translations/lang/en.json [hunk 1]
```diff
@@ -253,7 +253,7 @@
     "DEP_interface": "Generated Interface ",
     "NODE_Title": "Set Up Your Custom Node",
     "NODE_Subtitle": "To connect to a local node...",
-    "NODE_Warning": "Your node must be HTTPS in order to connect to it via MyCrypto.com. You can [download the MyCrypto repo & run it locally](https://github.com/MyCryptoHQ/MyCrypto/releases/latest) to connect to any node. Or, get free SSL certificate via [LetsEncrypt](https://letsencrypt.org/)",
+    "NODE_Warning": "Your node must be HTTPS in order to connect to it via MyCrypto.com. You can [download the MyCrypto repo & run it locally](https://github.com/MyCryptoHQ/mycrypto.com/releases/latest) to connect to any node. Or, get free SSL certificate via [LetsEncrypt](https://letsencrypt.org/)",
```
Comment (wbobeirne) [inline common/translations/lang/en.json]: We want to keep these URLs like this, even if they're not active right now. This repo will replace mycrypto.com when it launches.

File: common/translations/lang/en.json [hunk 2]
```diff
@@ -392,7 +392,7 @@
     "HELP_2a_Desc_5": "Print the wallet if you have a printer. Otherwise, write down your private key and address on a piece of paper. Store this as a secure location, separate from your computer and the USB drive. ",
     "HELP_2a_Desc_6": "Keep in mind, you must prevent loss of the keys and password due to loss or failure of you hard drive failure, or USB drive, or piece of paper. You also must keep in mind physical loss / damage of an entire area (think fire or flood). ",
     "HELP_2b_Title": "2b) How do I safely / offline / cold storage with MyCrypto? ",
-    "HELP_2b_Desc_1": "Go to [https://github.com/MyCryptoHQ/MyCrypto/releases/latest](https://github.com/MyCryptoHQ/MyCrypto/releases/latest). ",
+    "HELP_2b_Desc_1": "Go to [https://github.com/MyCryptoHQ/mycrypto.com/releases/latest](https://github.com/MyCryptoHQ/mycrypto.com/releases/latest). ",
```
Comment (wbobeirne) [inline common/translations/lang/en.json]: Same as above.

File: common/utils/consoleAdvertisement.ts [hunk 3]
```diff
@@ -2,26 +2,31 @@
...
+               NNmdhhyyyyyyyyhhddmN               
+            mdhyyyyyyyyyyyyyyyyyyyyyhdN           
+         mdyyyyyyyyyyyyyyyyyyyyyyyyyyyyhdN        
+      Ndhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhm      
+    Nmhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhm       
+    dyyyyyyyyyyys++oyyyyyyyyyys+/+syyyyyyyyyyhd   
+  Nhyyyyyyyyyy+.    .+yyyyyyo-     :syyyyyyyyyhdN          _       _         _    _     _ 
+ Nhyyyyyyyyyo.        .+yyo-         :yyyyyyyyyyd         | |     (_)       | |  | |   | |
+ hyyyyyyyyo.            .-             :yyyyyyyyhm        | | ___  _ _ __   | |  | |___| |
+myyyyyyyy.     -+oo/.        .+ooo:`    `oyyyyyyyh    _   | |/ _ \| | '_ \  | |  | / __| |
```
Comment (wbobeirne) [inline common/utils/consoleAdvertisement.ts]: I haven't run this yet so I might be wrong but it looks like there's an a backtick in here which is escaping the string. Does this work?

File: common/utils/consoleAdvertisement.ts [hunk 4]
```diff
@@ -2,26 +2,31 @@
...
+hyyyyyyy+    -syyyyyyo-    .oyyyyyyy/`   /yyyyyyyy   | |__| | (_) | | | | | | |__| \__ \_|
+yyyyyyyyy+::syyyyyyyyyyo::oyyyyyyyyyyy/:+yyyyyyyyy    \____/ \___/|_|_| |_|  \____/|___(_)
+yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
+yyyyyyyyyyyyyyyyysyyyyyyyyyyyyyysyyyyyyyyyyyyyyyyy    https://github.com/MyCrypto/mycrypto.com
```
Comment (wbobeirne) [inline common/utils/consoleAdvertisement.ts]: This should link to MyCryptoHQ/MyCrypto.

File: common/translations/lang/en.json [hunk 5]
```diff
@@ -619,7 +619,7 @@
     "ONBOARD_secure_1_content__1": "Phishers send you a message with a link to a website that looks just like MyCrypto, EtherDelta, Paypal, or your bank, but is not the real website. They steal your information and then steal your money.",
     "ONBOARD_secure_1_content__2": "Install [EAL](https://chrome.google.com/webstore/detail/etheraddresslookup/pdknmigbbbhmllnmgdfalmedcmcefdfn) or [MetaMask](https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn) or [Cryptonite by Metacert](https://chrome.google.com/webstore/detail/cryptonite-by-metacert/keghdcpemohlojlglbiegihkljkgnige) or the [MyCrypto Chrome Extension](https://chrome.google.com/webstore/detail/myetherwallet-cx/nlbmnnijcnlegkjjpcfjclmcfggfefdm) to block malicious websites.",
     "ONBOARD_secure_1_content__3": "Always check the URL: `https://www.mycrypto.com`.",
-    "ONBOARD_secure_1_content__4": "Always make sure the URL bar has `MYCRYPTO, LLC [US]` in green.",
+    "ONBOARD_secure_1_content__4": "Always make sure the URL bar has `MyCrypto, Inc (US)` in green.",
```
Comment (wbobeirne) [inline common/translations/lang/en.json]: Good catch, I think we also used MyCrypto LLC elsewhere in the codebase. We'll need to update more places with this.

File: common/utils/consoleAdvertisement.ts [hunk 6]
```diff
@@ -2,26 +2,31 @@
...
+    Nmhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyhhm       
+    dyyyyyyyyyyys++oyyyyyyyyyys+/+syyyyyyyyyyhd   
+  Nhyyyyyyyyyy+.    .+yyyyyyo-     :syyyyyyyyyhdN          _       _         _    _     _ 
+ Nhyyyyyyyyyo.        .+yyo-         :yyyyyyyyyyd         | |     (_)       | |  | |   | |
+ hyyyyyyyyo.            .-             :yyyyyyyyhm        | | ___  _ _ __   | |  | |___| |
+myyyyyyyy.     -+oo/.        .+ooo:      oyyyyyyyh    _   | |/ _ \| | '_ \  | |  | / __| |
+hyyyyyyy+    -syyyyyyo-    .oyyyyyyy/    /yyyyyyyy   | |__| | (_) | | | | | | |__| \__ \_|
+yyyyyyyyy+::syyyyyyyyyyo::oyyyyyyyyyyy/:+yyyyyyyyy    \____/ \___/|_|_| |_|  \____/|___(_)
+yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
+yyyyyyyyyyyyyyyyysyyyyyyyyyyyyyysyyyyyyyyyyyyyyyyy    https://github.com/MyCrypto/mycrypto.com
```
Comment (wbobeirne) [inline common/utils/consoleAdvertisement.ts]: This should be https://github.com/MyCryptoHQ/MyCrypto

File: common/translations/lang/en.json [hunk 7]
```diff
@@ -469,7 +469,7 @@
...
     "HELP_8_Desc_3": "You can import your unencrypted private key and your Geth/Mist Format (encrypted) files directly into geth / Ethereum Wallet / Mist very easily now. See question #12 below. ",
-    "HELP_8_Desc_4": "In addition, the likelihood of us taking MyCrypto down is slim to none. It costs us almost nothing to maintain as we aren't storing any information. If we do take the domain down, it still is, and always will be, publicly available at [https://github.com/MyCryptoHQ/MyCrypto](https://github.com/MyCryptoHQ/MyCrypto/tree/gh-pages). You can download the ZIP there and run it locally. ",
+    "HELP_8_Desc_4": "In addition, the likelihood of us taking MyCrypto down is slim to none. It costs us almost nothing to maintain as we aren't storing any information. If we do take the domain down, it still is, and always will be, publicly available at [https://github.com/MyCryptoHQ/mycrypto.com](https://github.com/MyCryptoHQ/mycrypto.com/tree/gh-pages). You can download the ZIP there and run it locally. ",
```
Comment (wbobeirne) [inline common/translations/lang/en.json]: Same as above

File: common/translations/lang/en.json [hunk 8]
```diff
@@ -553,8 +553,8 @@
...
-    "HELP_Sec_Desc_4": "Check the URL -- This site is being served through GitHub and you can see the source code here: [https://github.com/MyCryptoHQ/MyCrypto/tree/gh-pages](https://github.com/MyCryptoHQ/MyCrypto/tree/gh-pages) to [https://www.mycrypto.com](https://www.mycrypto.com). ",
-    "HELP_Sec_Desc_5": "For generating wallets, you can download the [source code and run it locally](https://github.com/MyCryptoHQ/MyCrypto/releases/latest). See #5 above. ",
+    "HELP_Sec_Desc_4": "Check the URL -- This site is being served through GitHub and you can see the source code here: [https://github.com/MyCryptoHQ/mycrypto.com/tree/gh-pages](https://github.com/MyCryptoHQ/mycrypto.com/tree/gh-pages) to [https://www.mycrypto.com](https://www.mycrypto.com). ",
+    "HELP_Sec_Desc_5": "For generating wallets, you can download the [source code and run it locally](https://github.com/MyCryptoHQ/mycrypto.com/releases/latest). See #5 above. ",
```
Comment (wbobeirne) [inline common/translations/lang/en.json]: Same as above

**Comment by clrxbl** [comment]
Changed GitHub repository links back to the MyCrypto repository & changed another instance of "MyCrypto LLC" to "MyCrypto,  Inc (US)"

**Pull request merged**